example/spec/parallel/ingestGranule/IngestGranuleWithDuplicateHandlingVersionSpec.js

```diff
@@ -238,12 +238,12 @@ describe('The IngestGranule workflow with DuplicateHandling="version" and a gran
       { stopOnError: false }
     ).catch(console.error);
 
+    await deleteGranule({ prefix, granuleId });
```

Comment:
Nit/non-blocking: Should be be try/caching this sort of deletion 'hoisting' and doing a console.error like what results from the other calls?      It's probably irrelevant as jasmine will print out AfterAll failures, but I hadn't really deeply thought about this pattern before now.